zerver/lib/external_accounts.py

```diff
@@ -23,18 +26,25 @@
     "twitter": {
         "text": "Twitter",
         "url_pattern": "https://twitter.com/%(username)s",
-        "name": "Twitter username",
+        "name": gettext_lazy("Twitter username"),
         "hint": "",
     },
     "github": {
         "text": "GitHub",
         "url_pattern": "https://github.com/%(username)s",
-        "name": "GitHub username",
+        "name": gettext_lazy("GitHub username"),
         "hint": "",
     },
 }
 
 
+def get_default_external_accounts() -> Dict[str, Dict[str, str]]:
+    DEFAULT_EXTERNAL_ACCOUNTS["twitter"]["name"] = str(DEFAULT_EXTERNAL_ACCOUNTS["twitter"]["name"])
+    DEFAULT_EXTERNAL_ACCOUNTS["github"]["name"] = str(DEFAULT_EXTERNAL_ACCOUNTS["github"]["name"])
```

Comment:
Two issues with this:
* You need to make a new object copying the original -- using, say, `copy.deepcopy` -- otherwise, this won't work after the first request
* We should loop over the first key, so that the code won't break if we add a new default external account.